### PR TITLE
Upgrade openjdk to 1.11

### PR DESCRIPTION
**Pull request by jchengsfx**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -255,13 +255,13 @@ RUN apt update &&\
 
 RUN add-apt-repository ppa:openjdk-r/ppa &&\
     apt update &&\
-    apt install -y openjdk-10-jdk maven
+    apt install -y openjdk-11-jdk maven
```
Comment (keitwb) [inline Dockerfile:258]: Make sure none of the development tools are going into the bundle.  It should just be the JRE with the core binaries and std library classes.

Comment (jchengsfx) [reply Dockerfile:258]: It'll currently has:
```drwxr-xr-x  1 root root 4096 Oct  6 21:14 bin
drwxr-xr-x  4 root root 4096 Oct  6 21:14 conf
drwxr-xr-x  3 root root 4096 Oct  6 21:14 docs
drwxr-xr-x  3 root root 4096 Oct  6 21:14 include
drwxr-xr-x  2 root root 4096 Oct  6 21:14 jmods
drwxr-xr-x 72 root root 4096 Oct  6 21:14 legal
drwxr-xr-x  6 root root 4096 Oct  6 21:14 lib
-rw-r--r--  1 root root 1161 Oct  6 21:14 release
```

Should everything except `bin` and `lib` be removed?

Comment (keitwb) [reply Dockerfile:258]: Not sure, make sure the tar bundle works on a system without any JRE installed.

Comment (keitwb) [reply Dockerfile:258]: Perferably one that isn't Ubuntu/Debian based either.

Comment (jchengsfx) [reply Dockerfile:258]: Alpine and centos are included in our bundle integration tests.

**Review by keitwb — APPROVED** [review]

**Pull request merged**